lib/td/command/list.rb

```diff
@@ -345,7 +345,7 @@ def self.finishup
   add_list 'connector:guess', %w[config?], 'Run guess to generate connector config file', ["connector:guess config.yml -o td-bulkload.yml\n\nexample config.yml:#{connector_guess_example_config}"]
   add_list 'connector:preview', %w[config], 'Show preview of connector execution', ['connector:preview td-bulkload.yml']
 
-  add_list 'connector:issue', %w[config], 'Run one time connector execution', ['connector:issue td-bulkload.yml']
+  add_list 'connector:issue', %w[config database table], 'Run one time connector execution', ['connector:issue example_db event_logs td-bulkload.yml']
```

Comment:
This change is not backward compatible. This makes existent automated shell scripts failed which are using --database and --table options.